src/app/pages/profil-parameter-page/profil-parameter-page.component.html

```diff
@@ -0,0 +1,170 @@
+<div class="background-image">
+    <div class="parameters-container">
+        <div class="top-parameters-container">
+             <img class="parameters-logo" src="/assets/pictures/parameters.png">
+            <h1>Paramètres</h1>
+        </div>
+
+        
+
+        <div class="profil-parameters-container">
+            <div class="profil-picture-container">
+                Avatar de profil
+                <div class="profil-picture">
+                    <div class="avatar">
+                        <img src="/assets/pictures/Connexion.png">
+                    </div>
+                    
+                    <button>changer d'Avatar</button>
```

Comment:
input and should be inside a form

Reply:
ici on ne va plus utiliser de photo de profil mais des avatars et le bouton dois ouvrir une pop-up je pense que le form doit être dans la pop-up